spec/person_spec.rb

```diff
@@ -0,0 +1,19 @@
+require './person'
+
+describe Person do
+  person = Person.new(21, 'John')
+
+  context 'should create instance of Person Class' do
+    it 'returns correct name' do
+      expect(person.name).to eq('John')
+    end
+
+    it 'returns correct age' do
+      expect(person.age).to eq(21)
+    end
+
+    it 'should return correct data' do
+      expect(person.can_use_services?).to eq true
+    end
```

Comment:
- [ ] it would be better to add tests for `add_rental` and also for `can_use_services` in order to cover all the methods for `can_use_services` you should test the positive case and negative case